Dockerfile-data-irods

```diff
@@ -0,0 +1,8 @@
+FROM mjstealey/docker-irods-icat:4.1.8
+
+ADD ./irods/delete_user.sh /home/hsuserproxy
+ADD ./irods/delete_user.sh /home/hsuserproxy
+
+RUN useradd -m -p hsuserproxy -s /bin/bash hsuserproxy
+
+RUN chmod -R 777 /home
```

Comment:
@cuahsimarko I believe line 3-6 are not needed here.

Reply:
Ok I added to my notes for subsequent optimizations